app/src/main/java/org/mozilla/focus/tiles/HomeTilesManager.kt

```diff
@@ -0,0 +1,50 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.tiles
+
+import android.content.Context
+import android.content.Context.MODE_PRIVATE
+import android.content.SharedPreferences
+import org.json.JSONArray
+import org.json.JSONObject
+
+const val HOME_TILES_PREFS = "hometilesPrefs"
+const val CUSTOM_SITES_LIST = "customSitesList"
+
+const val KEY_URL = "url"
+const val KEY_TITLE = "title"
+const val KEY_IMG = "img"
+const val KEY_ID = "identifier"
```

Comment:
nit: I think `id` is fine - lets save a few characters on disk.

Why is the `id` necessary anyway? Can URL be used as the ID instead? URLs are guaranteed to be unique so provided we never add the same URL twice, it should be sufficient though if the code is wrong and we get into a state where there are duplicated URLs, it could be problem...

Reply:
This is something that will be used by the custom tiles code too, and if those change, we don't want to be tied to a url.